release/release.go

```diff
@@ -1,3 +1,7 @@
+// Package release contains APIs for interacting with a
+// particular "release".  Avoid this unless you are sure
+// you need it.  In most cases it's expected
```

Comment:
s/In most cases//.  So far we've successfully avoided users having to look here.